addons/payment_asiapay/models/payment_provider.py

```diff
@@ -65,10 +73,9 @@ def _asiapay_get_api_url(self):
         """
         self.ensure_one()
 
-        if self.state == 'enabled':
-            return 'https://www.paydollar.com/b2c2/eng/payment/payForm.jsp'
-        else:  # 'test'
-            return 'https://test.paydollar.com/b2cDemo/eng/payment/payForm.jsp'
+        environment = 'production' if self.state == 'enabled' else 'staging'
```

Comment:
Is "staging" the dedicated term, or would "test" be ok too? I find "test" clearer.